frontend/src/components/dashboard/create/sms/TwilioCredentialsInput.tsx

```diff
@@ -27,36 +33,45 @@ const TwilioCredentialsInput = ({
     }
   }, [accountSid, apiKey, apiSecret, messagingServiceSid, onFilled])
 
-  return (
+  const renderLabel = (label: string, link: string) => (
```

Comment:
Was thinking if we can refactor this out as a separate component so we can consistently reuse it in the other credential inputs. For e.g. Telegram currently has this:

<img src='https://user-images.githubusercontent.com/3666479/88153816-80d68a00-cc38-11ea-8bac-e7ddb447d105.png' width=450 />